xla/service/gpu/buffer_comparator.cc

```diff
@@ -522,7 +522,8 @@ static const char* buffer_compare_ptx = R"(
 $L__BB1_9:
 	ret;
 
-}
+})"
+                                        R"(
```

Comment:
Isn't it better to add some comment in between in case someone try to merge it back...